dxvk.conf

```diff
@@ -339,6 +339,24 @@
 # environment variable is not set, otherwise it will be
 # ignored. The syntax is identical.
 
+# a valid value is : devinfo,fps,frametimes,pipelines,memory,version,api,compiler,
+
+#  here the different elements you can use 
+#    devinfo	: Displays the name of the GPU and the driver version.
+#    fps		: Shows the current frame rate.
+#    frametimes	: Shows a frame time graph.
+#    submissions: Shows the number of command buffers submitted per frame.
+#    drawcalls	: Shows the number of draw calls and render passes per frame.
+#    pipelines	: Shows the total number of graphics and compute pipelines.
+#    descriptors: Shows the number of descriptor pools and descriptor sets.
+#    memory		: Shows the amount of device memory allocated and used.
+#    gpuload	: Shows estimated GPU load. May be inaccurate.
+#    version	: Shows DXVK version.
+#    api		: Shows the D3D feature level used by the application.
+#    cs			: Shows worker thread statistics.
+#    compiler	: Shows shader compiler activity
+#    samplers	: Shows the current number of sampler pairs used [D3D9 Only]
+#    scale=x	: Scales the HUD by a factor of x (e.g. 1.5)
```

Comment:
```suggestion
# - devinfo: Displays the name of the GPU and the driver version.
# - fps: Shows the current frame rate.
# - frametimes: Shows a frame time graph.
# - submissions: Shows the number of command buffers submitted per frame.
# - drawcalls: Shows the number of draw calls and render passes per frame.
# - pipelines: Shows the total number of graphics and compute pipelines.
# - descriptors: Shows the number of descriptor pools and descriptor sets.
# - memory: Shows the amount of device memory allocated and used.
# - gpuload: Shows estimated GPU load. May be inaccurate.
# - version: Shows DXVK version.
# - api: Shows the D3D feature level used by the application.
# - cs: Shows worker thread statistics.
# - compiler: Shows shader compiler activity
# - samplers: Shows the current number of sampler pairs used [D3D9 Only]
# - scale=x: Scales the HUD by a factor of x (e.g. 1.5)
```

Reply:
I will integrate the presentation to the next commit 
do you want another pull request ? with your requested modification ?